edit/fix.go

```diff
@@ -396,6 +375,16 @@ func cleanUnusedLoads(f *build.File) bool {
 			all = append(all, load)
 		} else {
 			fixed = true
+			// If the load statement contains a post-comment, keep it
+			if len(load.Comment().After) == 0 {
+				continue
+			}
+
+			if len(all) == 0 {
```

Comment:
Why do you test len(all)?

Reply:
If `all` is not empty I can just attach the comments to the previous expression. My idea was that if there's an after-comment and we keep it, it belongs to the whole load-block and shouldn't be separated by an empty line: the following

    load(":a.bzl", "used")
    load(":b.bzl", "unused")
    # foo

becomes

    load(":a.bzl", "used")
    # foo

Reply:
I don't think the comment should be attached to another node. If we keep the comment, that means we consider it was incorrectly attached. So moving it to a block-comment might be better?

Reply:
Done.